### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by efb4f5ff-1298-471a-8973-3d47447115dc**
# Title

## Pull Request Type

- [ ] Bugfix
- [ ] Feature Implementation
- [ ] Documentation
- [ ] Other

## Desktop

- **OS:**
- **OS Version:**
- **FreeTube version:**

### CODE REVIEW

**Review by ChunkyProgrammer — CHANGES_REQUESTED** [review]
😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈😈

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@ChunkyProgrammer request changes again. Optimized some stuff.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
  <img src="https://docs.freetubeapp.io/images/logoColor.png" width=500 align="center">
 </p>
 
-FreeTube is an open source desktop YouTube player built with privacy in mind.
+FFreeTube is an open source desktop YouTube player built with privacy in mind.
```
Comment (ghost) [inline README.md]:
```suggestion
FFFreeTube is an open source desktop YouTube player built with privacy in mind.
```

**Review by ChunkyProgrammer — CHANGES_REQUESTED** [review]
Typo

**Pull request closed without merging**